config/software/omnibus-toolchain.rb

```diff
@@ -28,6 +28,9 @@
 # git!
 dependency "git"
 
+# fakeroot for EL based things
+dependency "fakeroot" if el?
```

Comment:
Is there any reason we wouldn't want this available in our toolchain for all platforms?